pyscf/adc/test/test_radc/test_H2O_radc_ip.py

```diff
@@ -89,12 +89,12 @@ def test_ea_adc3(self):
         self.assertAlmostEqual(e[0], 0.4481211042230935, 6)
         self.assertAlmostEqual(e[1], 0.5316292617891758, 6)
         self.assertAlmostEqual(e[2], 0.6850054080600295, 6)
-        self.assertAlmostEqual(e[3], 1.130637379907676, 6)
+        self.assertAlmostEqual(e[3], 1.1090318744878, 6)
```

Comment:
Why did these reference values change?

Reply:
There was a bug in the preconditioner for IP. I was getting different energy values using  adc(2) and adc(3) preconditioners. Fixed that.